content/md/products-super-power/enrich-products-variants.md

```diff
@@ -259,7 +259,7 @@ The product model and all its children (product models and variant products) are
 :::
 
 :::info
-It's not possible to mass delete product models from the product grid, because the delete of a product model could delete in cascade lots of children product models and variant products.
+Since the 2.2, you can also mass delete product models. **If you delete product models, all their children (product models and variant products) are also deleted.**
```

Comment:
```suggestion
Since the 2.2 version, you can also mass delete product models or delete product model directly from the product grid (by clicking on the trash icon). **If you delete product models, all their children (sub product models if any and they ariant products) will also be deleted.**
```

Reply:
Remove the "Since the 2.2 version", as you are merging in master it's the doc for the 3.x, we don't in which 2.x version it was introduced.
```suggestion
You can also mass delete product models. **If you delete product models, all their children (sub product models if any and they variant products) will also be deleted.**
```

Reply:
J'arrive trop tard, les filles ont déjà fait leurs commentaires ;)